src/TwigYard/Middleware/Error/ErrorMiddleware.php

```diff
@@ -81,11 +81,19 @@ public function __invoke(ServerRequestInterface $request, ResponseInterface $res
                 if (!ini_get('error_reporting')) {
                     return;
                 }
-                $this->loggerFactory->getLogger($this->appState->getUrl())
-                    ->critical(
-                        '[' . $request->getUri() . '] > ' . $errNo . ': ' . $errStr,
-                        $this->appState->dumpContext()
-                    );
+                try {
```

Comment:
Only the `$this->appState->getUrl()` should be in the `try` block. This will make it more readable.